jst-nuke-boilerplate

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Pulls Boilerplate into each package and attempts to push the changes up.
+# Copyright © 2017 RobLoach
+# MIT-licensed
+
+[ -f config.ini ] && \
+root=$(awk -F '=' '{if (! ($0 ~ /^;/) && $0 ~ /root/) print $2}' config.ini)
+
+cwd="`pwd`"
+
+for d in `echo ${root}jstransformer*/`; do
+  repo="`basename $d`"
+  echo "Updating $repo"
+  cd "$d"
+  rm -rf node_modules
+  git pull git@github.com:jstransformers/boilerplate.git master
+  git push origin master
```

Comment:
Agreed, I'd like it to check if there are merge conflicts, and then report them to a bad.txt or something.